### PR TITLE
docs: readme updates

### PR DESCRIPTION
**Pull request by meowgorithm**
This is a small readme housekeeping update.

* Simplify image
* Add badges
* Administer a few minor copy and punctuation edits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,12 +61,11 @@ melt restore ./recovered_id_ed25519 < words
 
 It all comes down to the private key __seed__:
 
-> Ed25519 keys start life as a 32-byte (256-bit) uniformly random binary seed (e.g. the output of SHA256 on some random input). The seed is then hashed using SHA512, which gets you 64 bytes (512 bits), which is then split into a “left half” (the first 32 bytes) and a “right half”. The left half is massaged into a curve25519 private scalar “a” by setting and clearing a few high/low-order bits. The pubkey is generated by multiplying this secret scalar by “B” (the generator), which yields a 32-byte/256-bit group element “A”.
-> https://blog.mozilla.org/warner/2011/11/29/ed25519-keys/
```
Comment (caarlos0) [inline README.md:64]: This link is missing after the edit

**Review by caarlos0 — COMMENTED** [review]
Just a small note, looks a loooot better though, thank you 🙏

File: README.md (hunk 1 above)
Comment (meowgorithm) [reply README.md:64]: Ah, so what I did there was use a markdown footnote. But if you think it's better the current way I can totally revert it back. Here's what it looks like:

* * *

Screen Shot 2022-03-17 at 9 11 44 PM

* * *

Screen Shot 2022-03-17 at 9 13 42 PM

Comment (caarlos0) [reply README.md:64]: aah I was reviewing through the phone and missed that, looks good then, dont mind my blindness  :D

**Pull request merged**